### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Pavlo000**
[DEMO LINK](https://Pavlo000.github.io/react_people-table-advanced/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,63 @@
...
 
-export default App;
+export const App: React.FC = () => {
+  const location = useLocation();
+
+  return (
+    <>
+      <nav
+        className="navbar is-fixed-top has-background-light"
+        data-cy="nav"
```
Comment (AndriiNychaiuk) [inline src/App.tsx:24]: Nice job! But let's make some improvements! I bet this navigation can be another component

File: src/components/NameFilter/NameFilter.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  return (
+    <p className="control has-icons-left">
+      <input
+        className="input"
+        type="text"
+        placeholder="Search"
+        value={query}
+        onChange={event => {
+          const newQuery = event.target.value;
+
```
Comment (AndriiNychaiuk) [inline src/components/NameFilter/NameFilter.tsx:16]: I think it's better to make a callback function for this purpose, because you have conditions here

**Review by AndriiNychaiuk — APPROVED** [review]

File: src/components/NameFilter/NameFilter.tsx (hunk 2 above)
Comment (mykola-koval) [reply src/components/NameFilter/NameFilter.tsx:16]: totally agree, let's make a callback here and in similar cases

File: src/components/CentryFilter/CentryFilter.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import { SearchLink } from '../SearchLink';
+
+export const CenturyFilter = () => {
+  const [searchParams] = useSearchParams();
+  const centuries = searchParams.getAll('centuries') || [];
+
+  return (
+    <div className="level is-flex-grow-1 is-mobile">
+      <div className="level-left">
+        {[16, 17, 18, 19, 20].map(String).map(century => (
```
Comment (mykola-koval) [inline src/components/CentryFilter/CentryFilter.tsx:12]: use a constant for the array

File: src/components/PeoplePage/PeoplePage.tsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        .toLocaleLowerCase()
+        .includes(lowerQuery);
+    });
+  }
+
+  if (sortField) {
+    visiblePeople.sort((a, b) => {
+      switch (sortField) {
+        case 'name':
+        case 'sex':
```
Comment (mykola-koval) [inline src/components/PeoplePage/PeoplePage.tsx:73]: use enum for cases

**Review by mykola-koval — APPROVED** [review]
good job 👍

**Pull request closed without merging**